wolfssl/openssl/asn1.h

```diff
@@ -16,4 +19,34 @@ struct WOLFSSL_ASN1_STRING {
     long flags;
 };
 
+
+WOLFSSL_API long wolfSSL_ASN1_INTEGER_get(const WOLFSSL_ASN1_INTEGER* i);
+WOLFSSL_API int wolfSSL_ASN1_INTEGER_get_int64(signed_word64 *p, const WOLFSSL_ASN1_INTEGER *i);
+
+#define ASN1_INTEGER WOLFSSL_ASN1_INTEGER
+#define int64_t signed_word64
+#define uint64_t word64
+
+#define ASN1_INTEGER_get wolfSSL_ASN1_INTEGER_get
+#define ASN1_INTEGER_get_int64 wolfSSL_ASN1_INTEGER_get_int64
+
+# define WOLFSSL_V_ASN1_NEG                      0x100
+# define WOLFSSL_V_ASN1_NEG_INTEGER              (2 | WOLFSSL_V_ASN1_NEG)
+# define WOLFSSL_V_ASN1_INTEGER                  2
+
+# ifndef INT64_MAX
+
+#  define __WOLFSSL_MAXUINT__(T) ((T) -1)
```

Comment:
This section of code needs to be changed.  Reference direct email from me for more details.

Reply:
Done